tests/config-ui/fields/test-class-config-field-advanced-settings.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * @package WPSEO\Tests\ConfigUI\Fields
+ */
+
+/**
+ * WPSEO_Config_Field_Advanced_Settings_Test
+ *
+ * @group configuration-wizard
+ */
+class WPSEO_Config_Field_Advanced_Settings_Test extends PHPUnit_Framework_TestCase {
+
+	/**
+	 * Tests the get data method.
+	 *
+	 * @dataProvider get_data_values
+	 *
+	 * @param string $expected     The expected result.
+	 * @param bool   $option_value The saved option value.
+	 */
+	public function test_get_data( $expected, $option_value ) {
```

Comment:
Please add an additional argument to the method to provide a description, which can be passed to the assertion as 3rd argument.